src/features/marketplace/store/marketplace.slice.spec.ts

```diff
@@ -45,4 +46,27 @@ describe('Marketplace slice', () => {
       expect(actual).toEqual(expected);
     });
   });
+
+  describe('When isListed is called', () => {
+    it('should return false from the state', () => {
+      const expected = false;
+      const actual = initialState.isListed;
```

Comment:
this test is not necessary, it is being covered by the first one (besides, this test is not testing the slice at all, it is just validating a variable value against another variable)